lib/inspec_tools/inspec.rb

```diff
@@ -304,11 +304,9 @@ def create_stig_data_element(attribute, control)
     def handle_severity(control)
       return if control[:impact].nil?
 
-      value = Utils::InspecUtil.get_impact_string(control[:impact])
+      value = Utils::InspecUtil.get_impact_string(control[:impact], false)
```

Comment:
Every time I see `get_impact_string` used I find myself wanting it to be a keyword argument so bad, I feel like it would be much more clear as one.